addons/osfstorage/models.py

```diff
@@ -560,3 +564,20 @@ def create_waterbutler_log(self, auth, action, metadata):
             auth=auth,
             params=params
         )
+
+    @property
+    def storage_usage(self):
+        storage_usage = cache.get('storage_usage' + self._id)
```

Comment:
Drive-by: we should probably separate the prefix from the variable part, e.g. `storage_usage:123ab` rather than `storage_usage123ab`. This will allow us to user `str.split` to get the GUID, if need be.